xbmc/platform/darwin/osx/XBMCApplication.mm

```diff
@@ -204,13 +139,41 @@ - (void)applicationDidFinishLaunching:(NSNotification*)note
 
   //window.acceptsMouseMovedEvents = TRUE;
   [NSApp activateIgnoringOtherApps:YES];
-  // kick our mainloop into an extra thread
-  [NSThread detachNewThreadSelector:@selector(mainLoopThread:) toTarget:self withObject:nil];
+
+  // kick our mainloop into a separate thread
+  //
```

Comment:
Ok.  As far as I can tell the code guidelines don't address this.
I use the blank line to make the comment above it to read like a paragraph title on a group of statements, rather than a comment applying only to the immediately following statement.
I suppose I can just put a blank line there without the comment marker.
Would that work, or can you suggest a better way to do this?

Reply:
Yeah a single empty line is fine imho. Personally I feel keeping the comment mark looks like a leftover